arrow-core-data/src/main/kotlin/arrow/typeclasses/Foldable.kt

```diff
@@ -241,7 +241,11 @@ interface Foldable<F> {
   fun <A> Kind<F, A>.firstOrNone(predicate: (A) -> Boolean): Option<A> =
     find { predicate(it) }
 
-  fun <A> Kind<F, A>.toList(): List<A> = foldRight(Eval.now(emptyList<A>())) { v, acc -> acc.map { listOf(v) + it } }.value()
+  /**
+   * Turn items into a list
+   */
```

Comment:
nit: I don't think the comment is adding much, so we could remove it, no?

```suggestion
```

Reply:
Agreed, it's also not really accurate. "Turning structure into list" would be better, but `toList` is descriptive enough imo.